com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/message/Messages.java

```diff
@@ -169,7 +169,8 @@
     public static final String ERROR_DURING_CLEAN_UP_0 = "Error during clean-up: {0}";
     public static final String COULD_NOT_DELETE_HISTORIC_PROCESS_0 = "Could not delete historic process \"{0}\"";
     public static final String COULD_NOT_ABORT_OPERATION_0 = "Could not abort operation \"{0}\"";
-
+    public static final String DETECTED_LEFTOVER_SERVICES = "Detected leftover services after undeploying application";

```

Comment:
Use something like "Services were not deleted as the --delete-services flag was not specified." That way the user would know what to use next time.